packages/lodestar/src/network/subnets/interface.ts

```diff
@@ -0,0 +1,20 @@
+import {Slot, ValidatorIndex} from "@chainsafe/lodestar-types";
+
+/** Generic CommitteeSubscription for both beacon attnets subs and syncnets subs */
+export type CommitteeSubscription = {
+  validatorIndex: ValidatorIndex;
+  subnet: number;
+  slot: Slot;
+  isAggregator: boolean;
+};
+
+export interface ISubnetsService {
+  start(): void;
+  stop(): void;
+  addCommitteeSubscriptions(subscriptions: CommitteeSubscription[]): void;
+  getActiveSubnets(): number[];
+}
+
+export interface IAttnetsService extends ISubnetsService {
```

Comment:
Either remove this interface or add a `ISyncnetsService` interface.
My preference is removing this interface (so that click thru navigates to the implementation and not just the interface, which only has a single implementation anyways)

Reply:
Agree with the concerns but let's discuss that in an upcoming issue